__tests__/components/forecast-summaries.test.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Enzyme from 'enzyme';
+import ForecastSummary from '../../src/components/forecast-summary';
```

Comment:
You wouldn't need to write `Enzyme.shallow()` if you used a named import. eg. `import { shallow } from 'enzyme'`